src/stlink-lib/chipid.c

```diff
@@ -98,44 +100,59 @@ void process_chipfile(char *fname) {
             } else if (strcmp(value, "L0_L1") == 0) {
                 ts->flash_type = STM32_FLASH_TYPE_L0_L1;
             } else if (strcmp(value, "L4_L4P") == 0) {
```

Comment:
If the configuration files now use L4 flash_type, then here you also need to change it to L4

Reply:
Thanks for reviewing! :smiley:
Ok, let me have a look...

Reply:
Ah, I see, you mean the `strcmp(value, "L4_L4P")` ,  yes this must be `strcmp(value, "L4")` of course...